src/app/select-trajectory/select-trajectory.page.ts

```diff
@@ -150,6 +159,70 @@ export class SelectTrajectoryPage {
     this.router.navigate(['/diary'])
   }
 
+  async createExample() {
+    this.settingsService.getConfig('newApp').subscribe(
```

Comment:
`'newApp'` could be defined in an enum within `SettingsService`. An enum (e.g. sth. like `SettingsConfig`) would be easier to reuse than a hardcoded string and can simply be expanded with other keys in the future.

Reply:
Added `SettingsConfig` to `SettingsService`.